engine/context/aci-integration.md

```diff
@@ -150,12 +150,20 @@ Name resolution between containers is achieved by writing service names in the `
 >
 > The current Docker Azure integration does not allow fetching a combined log stream from all the containers that make up the Compose application.
 
+## Updating applications
+
+From a deployed Compose application, you can update the application by re-deploying it with the same ACI group name: `docker compose up --project-name` with the Compose project name already used in previous deployments.
+
+Updating an application means the ACI node will be reused, and the application will keep the IP address already allocated to expose ports, if any. ACI also has some limitations on what can be updated in an existing application (you will not be able to change CPU/memory reservation for example), in these cases you need to deploy a new application from scratch.
+
+Updating is the default behaviour if you invoke several times `docker compose up` on the same compose file, since the Compose project name is derived from the compose file location by default. You need to explicitely execute `docker compose down` before running again `docker compose up` in order to totally reset a Compose application.
+
 ## Releasing resources
 
-Single containers and Compose applications can be removed from ACI with 
-the `docker prune` command. The `docker prune` command removes deployments 
+Single containers and Compose applications can be removed from ACI with

```

Comment:
Can't change this as a suggestion unfortunately but would be good to rephrase so that we say it's only stopped/terminated containers that are cleaned up by prune.

Reply:
It's actually any state that is not "running". ACI containers sometimes ends up in weird states that we can't all list. 
`The `docker prune` command removes deployments that are not currently running. ` is exactly what happens.

Reply:
OK then SGTM